tests/testthat/helper-for-tests.R

```diff
@@ -0,0 +1,11 @@
+loadTestSimulation <- function(simulationName) {
+  dataPath <- file.path(getwd(), "..", "data", fsep = .Platform$file.sep)
+  simFile <- file.path(dataPath, paste0(simulationName, ".pkml"), fsep = .Platform$file.sep)
+  sim <- loadSimulation(simFile)
+}
+
+executeWithTestFile <- function(actionWithFile) {

```

Comment:
This one is neat. It creates a temp file, passes it as a paraemter to the calling function and delete the file after the fact.

Reply:
Cool!